### DISCUSSION TITLE
Table rows splitting the content between pages

### DISCUSSION THREAD

**Discussion by rich1990** [mpdf/mpdf#1639] [Q&A] [answered]
I have the following issue:

<img width="346" alt="Screenshot 2022-04-08 at 15 59 04" src="https://user-images.githubusercontent.com/5670983/162451162-4a590bf7-1b85-485e-bf2c-efaeccb38beb.png">

Is it possible to split the row content between the 2 pages? i am using <tables> tags with page-break-inside: avoid - but it does not seems to have any effect.
Basically if i have a row with a lot of vertical content it goes in a new page, while theoretically should the white space remaining in the first page and complete the row in the second, so the first page would look like this:
<img width="554" alt="Screenshot 2022-04-08 at 16 05 22" src="https://user-images.githubusercontent.com/5670983/162452188-74021650-a759-4953-a413-abbd37d439fe.png">

**Comment by finwe** [comment, marked as answer]
No, mPDF will always place a row to a single page. As noted in the [documentation for tables](https://mpdf.github.io/tables/tables.html),

> Note: mPDF will always resize tables if it is the only way to fit a row or whole table onto a full page.

**Comment by rich1990** [comment]
Thanks, Do you know if someone is already working on this matter or if there is any future roadmap implementation for that?

**Reply by finwe** [reply]
Not likely. See the development notice in the readme of the project.

**Reply by rich1990** [reply]
Ok thanks
